website/docs: enhance blueprint docs (#15984)
goauthentik/authentik: website/docs/customize/blueprints/export.mdx

## Flow exports

> A specific flow with it's attached stages, policies, and other objects can be exported.

Reviewer: ```suggestion
A specific flow with its attached stages, policies, and other objects can be exported.
```

> This export can be triggered via the API or the Web UI by clicking the **Export** button (download icon) next to the flow in the flow list.

Reviewer: ```suggestion
This export can be triggered via the API or the Admin interface by clicking the **Export** button (download icon) next to the flow in the flow list.
```

> This export can be triggered via the API or the Web UI by clicking the **Export** button (download icon) in the flow list.

Reviewer: ```suggestion
This export can be triggered via the API or the Web UI by clicking the **Export** button (download icon) next to the flow in the flow list.
```
